server/api/controllers/harvestGames/get-hg-rankings.js

```diff
@@ -0,0 +1,39 @@
+module.exports = {
+  friendlyName: 'Get Harvest Games rankings',
+
+  description: 'Get Harvest Games rankings',
+
+  inputs: {
+    lgRankingId: {
+      required: false,
+      type: 'string',
+    },
+  },
+
+  exits: {
+    success: {
+      description: 'Harvest Games rankings returned successfully',
+    },
+    invalid: {
+      description: 'Failed to retrieve Harvest Games rankings',
+    },
+  },
+
+  fn: async function ({ lgRankingId }, exits) {
+    try {
+      if (lgRankingId) {
+        let data = await HGRankings.find({ _id: lgRankingId }).populateAll();
```

Comment:
don't think u need all these `populateAll` calls since you don't actually depend on other models. but if it breaks if you remove it, then i guess keep it